ca_on_peel/__init__.py

```diff
@@ -17,22 +17,26 @@ def get_organizations(self):
         organization.add_post(role="Mayor", label="Caledon", division_id="ocd-division/country:ca/csd:3521024")
         organization.add_post(role="Mayor", label="Brampton", division_id="ocd-division/country:ca/csd:3521010")
         organization.add_post(role="Mayor", label="Mississauga", division_id="ocd-division/country:ca/csd:3521005")
-        for ward_number in range(1, 6):
+        for ward_number in range(1, 7):
             organization.add_post(
                 role="Councillor",
-                label="Caledon Ward {}".format(ward_number),
+                label="Caledon Ward {} (seat 1)".format(ward_number),
                 division_id="ocd-division/country:ca/csd:3521024/ward:{}".format(ward_number),
             )
+        stop = 3
         for ward_number in range(1, 11):
-            organization.add_post(
-                role="Councillor",
-                label="Brampton Ward {}".format(ward_number),
-                division_id="ocd-division/country:ca/csd:3521010/ward:{}".format(ward_number),
-            )
+            if ward_number == 7:
+                stop = 2
+            for seat_number in range(1, stop):
+                organization.add_post(
+                    role="Councillor",
+                    label="Brampton Ward {} (seat {})".format(ward_number, seat_number),
+                    division_id="ocd-division/country:ca/csd:3521010/ward:{}".format(ward_number),
+                )
         for ward_number in range(1, 12):
             organization.add_post(
                 role="Councillor",
-                label="Mississauga Ward {}".format(ward_number),
+                label="Mississauga Ward {} (seat 1)".format(ward_number),
```

Comment:
Otherwise we would have to remove the seat numbers that get automatically added in `people.py` which would take a lot of code (similar to the huron scraper). Another option could be to use a variable for the seat number which is always 1 for those districts.

Reply:
Aha, there are [25 members](https://www.peelregion.ca/council/), but the CSV has 37 rows. That's because the same person at the regional level represents multiple wards at the municipal level. Except in Brampton, where two people represent the same ward for wards 1-6, but only 1 for 7-10.